TheMovieDB/.swiftlint.yml

```diff
@@ -15,10 +15,10 @@ excluded:
   - Pods
   - Project/R.generated.swift
 
-force_cast: error
-force_try: error
-force_unwrapping: error
-implicitly_unwrapped_optional: error
+force_cast: warning
+force_try: warning
+force_unwrapping: warning
+implicitly_unwrapped_optional: warning
```

Comment:
Why did you change that, it must be errors.

Reply:
https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/306178015c7980b8d5a5be341797b3c21ffb3e8d